libs/blocks/quiz-results/quiz-results.js

```diff
@@ -89,7 +89,7 @@ export default async function init(el, debug = null, localStoreKey = null) {
 
     loadFragments(el, basic);
   } else {
-    window.lana.log(`${LOADING_ERROR} The quiz-results block is misconfigured`);
+    window.lana?.log(`${LOADING_ERROR} The quiz-results block is misconfigured`);
```

Comment:
Maybe a `describe('Quiz Marquee (Biz Markie) doesn't have what you need)` could satisfy this by passing `'quiz-resutlss'` or some odd string?